### DISCUSSION TITLE
My two factor authentication stopped working on my mobile phone using GitHub app

### DISCUSSION THREAD

**Discussion by HowardCohl** [community/community#40946] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It used to work, but now it gives me a red box error code on the mobile app: 
```The operation couldn't be completed. (CryptoTokenKit error -3).```

Can someone please help me with this?

**Comment by ossanna16** [comment]
Hi there @HowardCohl and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by cx0y** [comment]
Hello @HowardCohl ! Are you using this application (Github)? If you're, then I'll recommend you, please don't use this application. Instead of this you can use [Google Authenticator](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2).
![Screenshot (98)](https://user-images.githubusercontent.com/93507446/206218196-98876e36-4fcd-408c-a33e-364ae34c40d5.png)

**Reply by HowardCohl** [reply]
Good idea @cx0y however it doesn't explain the error. I suspect this is a bug unless someone can tell me otherwise I'll report it.

**Reply by cx0y** [reply]
Yes it's possible that it's a bug! I used this application before, and it's kinda laggy to me.
